Ltac2 #[abstract] attribute for types (#18766)
coq/coq: doc/changelog/06-Ltac2-language/18766-ltac2-abstract-type.rst

- **Added:**

>   :attr:`abstract` for :cmd:`Ltac2 Type` makes it possible to turn types abstract at the end of a module

Reviewer: ```suggestion
  :attr:`abstract` attribute for :cmd:`Ltac2 Type` to turn types abstract at the end of the current module
```
